.github/workflows/main.yml

```diff
@@ -1,17 +1,32 @@
-name: CI
+name: meetup-talks
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v1
-    - name: Run a one-line script
-      run: echo Hello, world!
-    - name: Run a multi-line script
-      run: |
-        echo Add other actions to build,
-        echo test, and deploy your project.
+      - name: Setup linkstatus, mdspell and write-good
+        run: |
+          python3 -m pip install linkstatus --user
+          sudo apt install nodejs
+          sudo npm i markdown-spellcheck -g
+          sudo npm install write-good -g
```

Comment:
Instead of using `sudo npm ...` we should set the prefix with.
`npm config --global set prefix ${HOME}/.npm` 
And then run the commands with `~/.npm/bin/mdspell`

Let's have consistency with npm commands i.e. `npm install <package-name> -g`

Reply:
@bhavin192 Github Actions provides Linux and macOS VMs with passwordless `sudo`.
anyhow we need `sudo` for installing `nodejs`.

Yup, I love to fallow `npm` installation consistency.

Reply:
Yes, we will need sudo to install nodejs itself. But like we avoid sudo pip let's avoid sudo npm commands as well